File: C/1110.c
#include <stdio.h>

int main()
{
    int put, ten, one, new, newone;
    int tmp;
    int cnt = 0;

    scanf("%d", &put);
    tmp = put;

    while (1)
    {
        ten = put / 10;
        one = put % 10;
        newone = (ten + one) % 10;
        new = (10 * one) + newone;
        put = new;
        cnt++;
        if (tmp == put)
            break;
    }
    printf("%d", cnt);
    return 0;
}

File: Python/반복문/1110.py
first_put = int(input())
put = first_put
cnt = 0

while 1:
    ten = put // 10
    one = put % 10
    newone = (ten + one) % 10
    new = (10 * one) + newone
    put = new
    cnt += 1
    if put == first_put:
        break
print(cnt)
